controls/SV-244538.rb

```diff
@@ -81,20 +72,17 @@
 
   if virtualization.system.eql?('docker')
     impact 0.0
-    describe "Control not applicable within a container" do
-      skip "Control not applicable within a container"
+    describe 'Control not applicable within a container' do
+      skip 'Control not applicable within a container'
+    end
+  elsif package('gnome-desktop3').installed?
+    describe command('grep -i idle /etc/dconf/db/local.d/locks/*') do
+      its('stdout.split') { should include '/org/gnome/desktop/session/idle-delay' }
     end
   else
-    if package('gnome-desktop3').installed?
-      describe command("grep -i idle /etc/dconf/db/local.d/locks/*") do
-        its('stdout.split') { should include  '/org/gnome/desktop/session/idle-delay' }
-      end
-    else
-      impact 0.0
-      describe 'The GNOME desktop is not installed' do
-        skip 'The GNOME desktop is not installed, this control is Not Applicable.'
-      end
+    impact 0.0
+    describe 'The GNOME desktop is not installed' do
+      skip 'The GNOME desktop is not installed, this control is Not Applicable.'
     end
```

Comment:
This test set needs enhancement since other destkops are now becoming popular in the DOD. I updated the 'GNOME desktop check to see if any GUI desktop is installed in the xsystem but now we need to add the checks for other supported destkops'